tests/js/storyTelling-page-storyEditTests.js

```diff
@@ -67,6 +67,24 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
         }
     });
 
+    var expectedVisibility = {
+        prePublish: {
+            progressArea: false,
+            responseArea: false,
+            shareButton: false
```

Comment:
Why is the share button invisible before and after the publish?

Reply:
I see, it's the "disable" state rather than the visibility for the share button. Please ignore this.